parser/c2_parser.c2

```diff
@@ -189,9 +189,11 @@ fn void Parser.error(Parser* p, const char* format @(printf_format), ...) @(nore
     longjmp(&p.jmpbuf, 1);
 }
 
-fn void Parser.parseModule(Parser* p, bool is_generated) {
+fn void Parser.parseModule(Parser* p, bool is_interface, bool is_generated) {
     p.expectAndConsume(Kind.KW_module);
-    p.expectIdentifier();
+    // accept builtin types as module names
+    if (!(p.tok.kind >= Kind.KW_bool && p.tok.kind < Kind.KW_void))
```

Comment:
nicer to add function in token.c2 like is_keyword(), but then is_builtin_type()?

Reply:
Good point.